automatic/tortoisesvn/update.ps1

```diff
@@ -0,0 +1,74 @@
+import-module au
+import-module "$PSScriptRoot/../../extensions/chocolatey-core.extension/extensions/chocolatey-core.psm1"
+
+$releases = 'https://tortoisesvn.net/downloads.html'
+
+function global:au_BeforeUpdate {
+  Remove-Item "$PSScriptRoot\tools\*.exe"
+
+  $client = New-Object System.Net.WebClient
+  try 
+  {
+    $filePath32 = "$PSScriptRoot\tools\$($Latest.FileName32)"
+    $client.DownloadFile($Latest.URL32, "$filePath32")
+
+    $filePath64 = "$PSScriptRoot\tools\$($Latest.FileName64)"
+    $client.DownloadFile($Latest.URL64, "$filePath64")
+  }
+  finally 
+  {
+    $client.Dispose()
+  }
+
+  $Latest.ChecksumType = "sha256"
+  $Latest.Checksum32 = Get-FileHash -Algorithm $Latest.ChecksumType -Path $filePath32 | ForEach-Object Hash
+  $Latest.Checksum64 = Get-FileHash -Algorithm $Latest.ChecksumType -Path $filePath64 | ForEach-Object Hash
+}
```

Comment:
@AdmiringWorm Using `Get-RemoteFiles` here results in an exception:

```
tortoisesvn - checking updates using au version 2017.1.11.511
URL check
  https://sourceforge.net/projects/tortoisesvn/files/1.9.5/Application/TortoiseSVN-1.9.5.27581-win32-svn-1.9.5.msi/download
  https://sourceforge.net/projects/tortoisesvn/files/1.9.5/Application/TortoiseSVN-1.9.5.27581-x64-svn-1.9.5.msi/download
nuspec version: 0.0
remote version: 1.9.5.27581
New version is available
Automatic checksum skipped
Running au_BeforeUpdate
Purging msi/download
Downloading to TortoiseSVN-1.9.5.27581-win32-svn-1.9.5.msi_x32.msi/download - https://sourceforge.net/projects/tortoisesvn/files/1.9.5/Application/TortoiseSVN-1.9.5.27581-win32-svn-1.9.5.msi/download
Exception calling "DownloadFile" with "2" argument(s): "An exception occurred during a WebClient request."
At C:\Program Files\WindowsPowerShell\Modules\au\Public\Get-RemoteFiles.ps1:63 char:13
+             $client.DownloadFile($Latest.URL32, $file_path)
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : WebException
```

Any idea? Has `Get-RemoteFiles` problems with the redirect on the page?